From 997e1423b762922d1ed7b21379ac9e4a76184ff0 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 11 Jan 2013 16:56:45 +0100
Subject: [PATCH] Add pkg-config file

For modern Linux distributions and development to use.

The .pc file also exposes the "databasedir" variable for a default
location to load the databases from, for use by application
developers.
---
 Makefile.am  |  5 ++++-
 configure.in |  1 +
 geoip.pc.in  | 11 +++++++++++
 3 files changed, 16 insertions(+), 1 deletion(-)
 create mode 100644 geoip.pc.in

diff --git a/Makefile.am b/Makefile.am
index bd4da3a..746ac6e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,4 +8,7 @@ SUBDIRS =		\
 	test		\
 	man
 
-EXTRA_DIST = README.OSX READMEwin32static.txt README.MinGW READMEwin32.txt GeoIPWinDLL.patch TODO bootstrap GeoIP.spec GeoIP.spec.in Makefile.netware Makefile.vc Makefile.win32 get_ver.awk geoip.ico
+pkgconfigdir = $(libdir)/pkgconfig/
+nodist_pkgconfig_DATA = geoip.pc
+
+EXTRA_DIST = README.OSX READMEwin32static.txt README.MinGW READMEwin32.txt GeoIPWinDLL.patch TODO bootstrap GeoIP.spec GeoIP.spec.in Makefile.netware Makefile.vc Makefile.win32 get_ver.awk geoip.ico geoip.pc.in
diff --git a/configure.in b/configure.in
index 77309b1..971b4aa 100644
--- a/configure.in
+++ b/configure.in
@@ -41,6 +41,7 @@ AC_CHECK_FUNC(gethostbyname_r, [
 AC_OUTPUT([
 Makefile
 GeoIP.spec
+geoip.pc
 libGeoIP/Makefile
 apps/Makefile
 conf/Makefile
diff --git a/geoip.pc.in b/geoip.pc.in
new file mode 100644
index 0000000..d943880
--- /dev/null
+++ b/geoip.pc.in
@@ -0,0 +1,11 @@
+prefix=@prefix@
+libdir=@libdir@
+includedir=@includedir@
+datadir=@datarootdir@
+
+Name: geoip
+Description: A non-DNS IP-to-country resolver library.
+Version: @VERSION@
+Libs: -L${libdir} -lGeoIP
+Cflags: -I${includedir}/
+databasedir=${datadir}/GeoIP
-- 
1.8.0.1

